tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Test_DampedHarmonicGaugeQuantities.cpp

```diff
@@ -0,0 +1,3180 @@
+                                            spatial_metric, d_spatial_metric);
+  const auto pi = GeneralizedHarmonic::pi(
+      lapse, dt_lapse, shift, dt_shift, spatial_metric, dt_spatial_metric, phi);
+
+  // commonly used terms
+  const auto exp_fac_1 = 1. / 2.;
+  const auto& log_fac_1 =
+      GeneralizedHarmonic::DampedHarmonicGauge_detail::log_factor_metric_lapse<
+          DataVector>(lapse, sqrt_det_spatial_metric, exp_fac_1);
+
+  // Initialize settings
+  const double amp_coef_L1 = pdist(generator);
+  const int exp_L1 = idist(generator);
+  const double t_start_L1 = pdist(generator) * 0.1;
+  const double sigma_t_L1 = pdist(generator) * 0.2;
+  const double r_max = pdist(generator) * 0.7;
+
+  // Tempering functions
+  const double roll_on_L1 =
+      GeneralizedHarmonic::DampedHarmonicGauge_detail::roll_on_function(
+          t, t_start_L1, sigma_t_L1);
+  const auto d0_roll_on_L1 = GeneralizedHarmonic::DampedHarmonicGauge_detail::
+      time_deriv_of_roll_on_function(t, t_start_L1, sigma_t_L1);
+  const auto& weight =
+      GeneralizedHarmonic::DampedHarmonicGauge_detail::weight_function<
+          SpatialDim, Frame::Inertial, DataVector>(x, r_max);
+  auto d4_weight = GeneralizedHarmonic::DampedHarmonicGauge_detail::
+      spacetime_deriv_of_weight_function<SpatialDim, Frame::Inertial,
+                                         DataVector>(x, r_max);
+
+  // coeffs that enter gauge source function
+  const auto mu_L1 =
+      amp_coef_L1 * roll_on_L1 * get(weight) * pow(get(log_fac_1), exp_L1);
+
+  // Calc \f$ \mu_1 = \mu_{L1} log(rootg/N) = R W log(rootg/N)^5\f$
+  const auto mu1 = mu_L1 * get(log_fac_1);
+
+  // Calc \f$ \partial_a [R W] \f$
+  auto d4_RW_L1 = d4_weight;
+  for (size_t a = 0; a < SpatialDim + 1; ++a) {
+    d4_RW_L1.get(a) *= roll_on_L1;
+  }
+  d4_RW_L1.get(0) += get(weight) * d0_roll_on_L1;
+
+  // Calc derivs of \f$ \mu_1 \f$
+  auto d4_mu1 =
+      make_with_value<tnsr::a<DataVector, SpatialDim, Frame::Inertial>>(
+          get(lapse), 0.);
+  {
+    const auto& d4_log_fac_mu1 =
+        GeneralizedHarmonic::DampedHarmonicGauge_detail ::
+            spacetime_deriv_of_power_log_factor_metric_lapse<
+                SpatialDim, Frame::Inertial, DataVector>(
+                lapse, shift, spacetime_unit_normal, inverse_spatial_metric,
+                sqrt_det_spatial_metric, dt_spatial_metric, pi, phi, exp_fac_1,
+                exp_L1 + 1);
+    for (size_t a = 0; a < SpatialDim + 1; ++a) {
+      // \f$ \partial_a \mu_1 \f$
+      d4_mu1.get(a) =
+          amp_coef_L1 * pow(get(log_fac_1), exp_L1 + 1) * d4_RW_L1.get(a) +
+          amp_coef_L1 * roll_on_L1 * get(weight) * d4_log_fac_mu1.get(a);
+    }
+  }
+
+  // Calc \f$ \partial_a N = {\partial_0 N, \partial_i N} \f$
+  auto d4_N = make_with_value<tnsr::a<DataVector, SpatialDim, Frame::Inertial>>(
+      get(lapse), 0.);
+  d4_N.get(0) = get(dt_lapse);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    d4_N.get(1 + i) 
```

Comment:
Remove these `TimeOut` changes. I think they are not necessary. 
You should reduce the number of grid points, if you think the time of computation will exceed the limit.

Reply:
removed